crates/cli/benches/benchmark.rs

```diff
@@ -46,9 +46,9 @@ impl FunctionCase {
     ) -> Result<Self> {
         let name = function_dir
             .file_name()
-            .ok_or(anyhow!("Path terminates in .."))?
+            .ok_or_else(|| anyhow!("Path terminates in .."))?
```

Comment:
Hmm, my guess would be that you're running a different version of Rust locally compared to what's running in CI. CI _should_ be using Rust's stable toolchain based on the `rust-toolchain.toml` file but it's possible something has gone awry.

Reply:
Looks like the `or_fun_call` check was is no longer enabled by default in Clippy as of Rust 1.67 https://github.com/rust-lang/rust-clippy/blob/master/CHANGELOG.md#moves-and-deprecations-1 so this would be flagged by Clippy if you're running a Rust version locally older than 1.67.